docs/contrib/modifications.md

```diff
@@ -4,6 +4,21 @@ You have two options for editing content: directly in your browser
 using GitHub, or using a Git-based workflow from your local work
 environment.
 
+## Notes on specific content additions
+
+**Screen shots:** If you are contributing a change that contains
+screen shots from {{extra.gui}}, set your screen resolution to 1080p
+(1920×1080 pixels) before you start creating them.
```

Comment:
That is understandable, but I believe it's a bit unreasonable to ask anyone to change their screen resolution (which, we can safely assume, is already optimal for the user's monitor). What we can ask users to do, though, is to _resize_ their screenshots down to 1080p or close to that _before_ uploading.